proteome-rest/src/main/java/org/uniprot/api/proteome/service/ProteomeQueryService.java

```diff
@@ -40,6 +44,17 @@ public ProteomeQueryService(
         this.queryProcessor = proteomeQueryProcessor;
     }
 
+    @Override
+    protected SolrRequest.SolrRequestBuilder createSolrRequestBuilder(
+            BasicRequest request,
+            AbstractSolrSortClause solrSortClause,
+            SolrQueryConfig queryBoosts) {
+        SolrRequest.SolrRequestBuilder builder =
+                super.createSolrRequestBuilder(request, solrSortClause, queryBoosts);
+        builder.filterQuery(getActiveProteomeFilterQuery());
+        return builder;
+    }
+
```

Comment:
We can do it later. But i feel, the value of argument(EXCLUDED_FIELD + ":false") in method builder.filterQuery should come from BasicRequest object. That way the service will be independent of UI business logic. If someone else can use our service class without filtering by excluded.